File: 438-C.c
#include <iostream>
#include <cstring>
using namespace std;

int main() {
	int n, k, temp;
	int vis[20];
	memset(vis, 0, sizeof(vis));
	cin>> n >> k;
	for(int i = 0; i < n; i++) {
		int ans = 0;
		for(int j = 0; j < k; j++){
			cin >> temp;
			ans = ans *2 + temp;
		}
		vis[ans]++;
	}
	for(int i = 0; i < (1<<k); i++) {
		for(int j = 0; j < (1<<k); j++) {
			if(vis[i] > 0 && vis[j] > 0 && (i&j) == 0) {
				cout<< "YES" << endl;
				return 0;
			}
		}
	}
	cout << "NO" << endl;
	return 0;
}

File: 438-C.go
package main

import (
	"fmt"
)

var n int
var k int
var temp int
var vis [200]int

func main() {
	fmt.Scan(&n, &k)
	for i := 0; i < n; i++ {
		ans := 0
		for j := 0; j < k; j++ {
			fmt.Scan(&temp)
			ans = ans*2 + temp
		}
		vis[ans]++
	}
	for i := 0; i < (1 << uint(k)); i++ {
		for j := 0; j < (1 << uint(k)); j++ {
			if vis[i] > 0 && vis[j] > 0 && ((i & j) == 0) {
				fmt.Println("YES")
				return
			}
		}
	}
	fmt.Println("NO")
}
